src/utils/cpm.js

```diff
@@ -2,9 +2,15 @@ import {auctionManager} from '../auctionManager.js';
 import {bidderSettings} from '../bidderSettings.js';
 import {logError} from '../utils.js';
 
-export function adjustCpm(cpm, bidResponse, bidRequest, {index = auctionManager.index, bs = bidderSettings} = {}) {
+export function adjustCpm(cpm, bidResponse, bidRequest, {index = auctionManager.index, bs = bidderSettings} = {}, gba = getBidAdjustment) {
   bidRequest = bidRequest || index.getBidRequest(bidResponse);
-  const bidCpmAdjustment = bs.get(bidResponse?.bidderCode || bidRequest?.bidder, 'bidCpmAdjustment');
+
+  const adapterAllowAlternateBidderCodes = bs.get(bidResponse?.adapterCode || bidRequest?.bidder, 'allowAlternateBidderCodes');
+  const catchUnknownBidderCodesWithAdapterBidAdjustment = bs.get(bidResponse?.adapterCode || bidRequest?.bidder, 'catchUnknownBidderCodesWithAdapterBidAdjustment');
```

Comment:
1. this setting (and variable) name is way too long, past the point of ridiculous. I'll offer `adjustAlternateBids`, or `adjustAllBids`.
2. what's the reason for looking at `allowAlternateBidderCodes`? If you have a bid from an alternate code (that is, one where `adapterCode` is different from `bidderCode`), it should mean that it was allowed already. 
3. I can't make sense of the fallback `bidResponse?.adapterCode || bidRequest?.bidder`. If you are using `bidRequest?.bidder`, are you not just using the "normal" bid adjustment rules?